app/views/manuals/show.html.erb

```diff
@@ -23,10 +23,15 @@
     presented_document: presented_document
   )
 %>
-<%=
-  render(
-    'manual_section',
-    presented_manual: presented_manual,
-    presented_document: presented_document
-  )
-%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
```

Comment:
Based on looking at a manuals page, I wonder if this should be from-tablet rather than from-desktop? eg https://manuals-fron-two-thirds-jnnriy.herokuapp.com/hmrc-internal-manuals/holding-and-movements-assurance-guidance/hmag60320

I suggest going to the national archives or archive.org to confirm what it used to do and matching if you didn't already (apologies if you did!).

Reply:
It appears that govspeak is applying it's own width wrapper for HMRC manuals 😬  I'll amend this to not include HMRCs at the very least and investigate what the original implementation was.

Reply:
Ah, that's not quite right. There's a custom width wrapper in the markup itself for HMRC manuals. I'll just scoop that out then, easy.